README.md

```diff
@@ -1,13 +1,22 @@
 # Project Express API
 
-Replace this readme with your own information about your project.
-
-Start by briefly describing the assignment in a sentence or two. Keep it short and to the point.
+Create an express API with at least 2 routes that return single result or an array of elements.
 
 ## The problem
 
-Describe how you approached to problem, and what tools and techniques you used to solve it. How did you plan? What technologies did you use? If you had more time, what would be next?
+My first basic backend-project. 
+I created the api by using a database which was given as an example for the project. It was originally found in kaggle.com, converted from csv to json through csvjson.com. I used postman for testing and learned how to create the API from all given video- and livesession material.
+
+If I had more time I would create a frontend page and more relevant routes.
 
 ## View it live
 
-Every project should be deployed somewhere. Be sure to include the link to the deployed project so that the viewer can click around and see what it's all about.
+https://project-express-api-bxmnrnoyjq-lz.a.run.app/
```

Comment:
Thank you for having the endpoints displayed like this and in the pull request, it was easy to find for me :)